### PR TITLE
[INFRA/IIGO] Judge changes now called from orchestration

### PR DESCRIPTION
**Pull request by Neelesh99**
# Summary

Calling new sanctions functions in the judiciary from the orchestration file.

## Additional Information

Had to adjust function signatures and test to fit new datatype being used.

## Test Plan

Adjusted test for new datatype.

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

File: internal/server/iigointernal/judiciary.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 // to be moved to paramters
 const sanctionCacheDepth = 3
 const historyCacheDepth = 3
+const maxNoReport = 500
 
 // to be changed
 const sanctionLength = 2
```
Comment (lhl2617) [inline internal/server/iigointernal/judiciary.go:21]: when?

File: internal/server/iigointernal/judiciary.go [hunk 2]
```diff
@@ -299,27 +300,32 @@ func createBroadcastsForRuleViolationPenalties(penalties map[string]roles.IIGOSa
...
+						VariableName: rules.ConstSanctionAmount,
+						Values:       []float64{0},
+					},
+					rules.TurnsLeftOnSanction: {
+						VariableName: rules.TurnsLeftOnSanction,
+						Values:       []float64{float64(sanction.TurnsLeft)},
+					},
+				})
+				totalSanctionPerAgent[sanction.ClientID] += sanctionVal
+			} // When logger PR is available, pass through here and log the missing sanction
```
Comment (lhl2617) [inline internal/server/iigointernal/judiciary.go]:
```suggestion
			} // TODO: When logger PR is available, pass through here and log the missing sanction
```

File: internal/server/iigointernal/judiciary.go (hunk 1 above)
Comment (Neelesh99) [reply internal/server/iigointernal/judiciary.go:21]: Yup, raising an issue and will address in next PR (Issue #262 )

File: internal/server/iigointernal/judiciary.go (hunk 2 above)
Comment (Neelesh99) [reply internal/server/iigointernal/judiciary.go]: Thanks

**Review by lhl2617 — APPROVED** [review]
lgtm

**Pull request merged**